### ISSUE TITLE
Integrity Error after publishing when upgraded to use VS 2019 Preview 1.0

### ISSUE THREAD

**Issue by nssidhu** [dotnet/core#6392]
I am using blazor wasm hosted app with .NET 5

I have been publishing it on daily basis without any error to Azure App Service (windows).
I have been using VS 2019 Version 16.10.1

Today when i instead started using using Microsoft Visual Studio 2019 Preview Version 16.11.0 Preview 1.0,  (Installed side-by-side on same machine)

Everything worked fine on the development box, but after publishing I  started getting bunch of Integrity Errors and the Website in production is totally down.

I am clueless as to what might have changed and how to go about resolving this ?

```
Failed to find a valid digest in the 'integrity' attribute for resource 'https://XXXXXXX.net/_framework/Blazored.FluentValidation.dll' with computed SHA-256 integrity 'aNNhqrhmkKK9ERGBVZ+w421s6OxC9NQVtEso5+PbF8k='. The resource has been blocked.

Failed to find a valid digest in the 'integrity' attribute for resource 'https://XXXXXXX.net.net/_framework/Blazored.LocalStorage.dll' with computed SHA-256 integrity '+/IJo9wLK9sTvL8ee9W9w8S1Gkj8rsf/6yob0uUFAaM='. The resource has been blocked.

many more such errors..........................
```

For the time being I reverted back to VS 2019 VS 2019 Version 16.10.1 and it is publishing correctly.
But want to understand what thing will i have to update to continue using existing Blazor Hosted WebAssembly with newer version of Visual studio ?

**Comment by nssidhu** [comment]
Found out that Error are also appearing for the VS 2019 (non Preview Version as well)

**Comment by nssidhu** [comment]
Since the error only happens for the version published to Azure App Service(Local one works fine), i suspect there are some reminent files on the Azure App Service.
What is the correct way to clear the files on Azure App Service (Windows) ?
Also i noticed that in the properties
(Stack settings
Stack
.NET version )
 of Azure App Service, after publishing it reverts it back to 3.1 
FYI: I upgraded my blazor wasm hosted app from 3.1 to .NET 5.0

**Comment by nssidhu** [comment]
Found the problem, after i opened in VS 2019 preview 2 and published it. did not change anything related to version it is still .NET 5
No matter, even if open in the prevision VS 2019 (non Preview Version) it would still published .NET 6 preview version of .js files under the _framework.

Not sure that change go applied, I uninstalled VS 2019 Preview and an the regular VS 2019.
Later just installed the VS 2019(non preview version) and it now publishes correctly
